changelog/front-commerce-3.9-2.34.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: "Release: 3.9 & 2.34"
+authors: pierre
+date: 2024-11-21T11:11
+hide_table_of_contents: true
+---
+
+## Consent Mode V2, Consistent Price Display, Gezy advanced capabilities and Extension Features API
+
+We know you're likely busy preparing for the Black Friday and the holiday
+season, so do we! 🎄 \
+For the last release of the year, we managed to deliver a handful of new
+features and improvements in Front-Commerce 3.9 and 2.34.
```

Comment:
I'm not a big fan of the "managed to deliver", I would suggest more something in the lines of:

```suggestion
For our final release of the year, we successfully introduced several new features and enhancements in Front-Commerce versions 3.9 and 2.34.
```